tests/UnitTest.php

```diff
@@ -45,7 +48,11 @@ public function testCreateMediaWithSetPreviewDrivers(): void
             hash: PreviewDriverEnum::Hash
         );
 
+        Storage::assertExists('preview_file.png');
+
         $this->assertEqualsFixture('create_media_with_set_preview_drivers.json', $media->toArray());
+
+        self::$mediaTestState->assertChangesEqualsFixture('create_media_with_set_preview_drivers.json', true);
```

Comment:
please remove json extension from other tests as well

```suggestion
        self::$mediaTestState->assertChangesEqualsFixture('create_media_with_set_preview_drivers');
```